Sources/Responses/Data+BytesRepresentable.swift

```diff
@@ -4,4 +4,11 @@ extension Data: BytesRepresentable {
   public var toBytes: [UInt8] {
     return [UInt8](self)
   }
+
+}
+
+extension Data {
+  // public static func == (lhs: BytesRepresentable, rhs: Data) -> Bool {
+  //   return lhs.toBytes == rhs.toBytes
+  // }
```

Comment:
Another attempt to extend equality operator to type that conforms to BytesRepresentable.